examples/C++/DDS/ContentFilteredTopicExample/ContentFilteredTopicExamplePublisher.hpp

```diff
@@ -45,35 +50,46 @@ class ContentFilteredTopicExamplePublisher : public eprosima::fastdds::dds::Data
     bool publish(
             bool waitForListener = true);
```

Comment:
Nit: Coding style for variables is camel case
```suggestion
            bool wait_for_listener = true);
```

Reply:
Not addressed in the header